frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -31,15 +31,20 @@ common_settings.network = {
     text = _("Network"),
     sub_item_table = NetworkMgr:getMenuTable()
 }
+
 common_settings.screen = {
     text = _("Screen"),
     sub_item_table = {
         require("ui/elements/screen_dpi_menu_table"),
         require("ui/elements/screen_eink_opt_menu_table"),
         require("ui/elements/screen_disable_double_tap_table"),
-        require("ui/elements/refresh_menu_table"),
+        require("ui/elements/refresh_menu_table")
```

Comment:
Please note that adding a comma at the end of the last element of a table is a convention especially with each element in one line. The reason is that if we add another element `git diff` only shows 1 line of change.